contributors.md

```diff
@@ -0,0 +1,26 @@
+# Contributors
+This project is in collaboration with [Utrecht Applied Data Science](https://www.uu.nl/en/research/applied-data-science),
+[Information and Technology Services](https://www.uu.nl/en/organisation/information-and-technology-services-its), and [Utrecht University Library](https://www.uu.nl/en/university-library).
+
+- [Rens van de Schoot](https://www.rensvandeschoot.com) (Main coordinator, [@Rensvandeschoot](https://github.com/Rensvandeschoot))
+- [Daniel Oberski](https://www.uu.nl/staff/DLOberski) (Scientific Director, [@daob](https://github.com/daob)).
+-  [Jonathan de Bruin](https://www.uu.nl/staff/JdeBruin1) (Lead engineer, [@J535D165](https://github.com/J535D165))
+- Raoul Schram ([@qubixes](https://github.com/qubixes)).
+- [Parisa Zahedi](https://www.linkedin.com/in/parisa-zahedi-28b17148/) ([@parisa-zahedi](https://github.com/parisa-zahedi))
+- [Kees van Eijden](https://www.uu.nl/staff/CJvanEijden) ([@KvEijden](https://github.com/KvEijden))
+- [Jan de Boer](https://www.uu.nl/staff/JdeBoer)
+- [Edu Hackenitz](https://www.uu.nl/staff/EJMHackenitz)
+- [Felix Weijdema](https://www.uu.nl/staff/FPWeijdema)
+- [Bianca Kramer](https://www.uu.nl/staff/BMRKramer)
+- [Pim Huijnen](https://www.uu.nl/staff/PHuijnen) (Digital Cultural History at the Department of History and Art History)
+- [Lars Tummers](https://www.uu.nl/staff/LGTummers) (Public Management and Behavior at Utrecht University, School of Governance)
+- Gerbrich Ferdinands ([@GerbrichFerdinands](https://github.com/GerbrichFerdinands))
+- [Qixiang Fang](https://www.uu.nl/staff/QFang) ([@fqixiang](https://github.com/fqixiang))
+- Albert Harkema ([@sasafrass](https://github.com/sasafrass))
+- Joukje Willemsen ([@JoukjeWillemsen](https://github.com/JoukjeWillemsen))
+- Maarten Hoogerwerf
+- Maarten Meijer ([@mmeijer23](https://github.com/mmerijer23))
+- D W J Visser
+- Martijn Huijts ([@MartijnUX](https://github.com/MartijnUX))
+- Sybren Hindriks
+- Yongchao Terry Ma ([@terrymyc]([@MartijnUX](https://github.com/terrymyc))
```

Comment:
There's an extra Martijn here.

Reply:
@GerbrichFerdinands can you check with all persons if they want to be mentioned here?

Reply:
Great remark, I will send everyone an e-mail today!